packages/overlay/src/popper.ts

```diff
@@ -13,17 +13,19 @@ governing permissions and limitations under the License.
 // Bundle only what we want from popper
 // See: https://popper.js.org/docs#popper-lite-tree-shaking
 
+// eslint-disable-next-line @typescript-eslint/triple-slash-reference
+/// <reference path="../local.d.ts" />
```

Comment:
This is the weirdest part. Even though I could get this to connect via `yarn tsc --build packages/overlay/tsconfig.json` without this reference, I could never get `yarn tsc --build packages/*/tsconfig.json`, our standard usage, to work without this... 😞